src/routes/Index.jsx

```diff
@@ -4,12 +4,17 @@ import NotFound from '../components/404';
 import Login from '../components/auth/Index';
 import Article from '../views/article/Index';
 import LandingPage from '../views/landingPage/Index';
+import RequestPasswordReset from '../views/passwordReset/RequestPasswordReset';
+import ResetPassword from '../views/passwordReset/ResetPassword';
 
 const Routes = () => (
   <Switch>
+    <Route path="/" exact component={LandingPage} />
     <Route path="/login" exact component={Login} />
     <Route path="/articles/:slug" component={Article} />
-    <Route path="/" exact component={LandingPage} />
+    <Route path="/articles/:slug" component={Article} />
```

Comment:
@youngestdj why is this route declared twice?

Reply:
Fixed. @9jaswag